docker/zuul/Dockerfile

```diff
@@ -23,24 +23,24 @@ RUN  git config --global user.email "zuul@otc-service.com" && \
 
 RUN git clone https://opendev.org/zuul/zuul zuul_src && \
     cd /zuul_src && \
-    git pull https://review.opendev.org/zuul/zuul refs/changes/40/859940/10
+    git pull https://review.opendev.org/zuul/zuul refs/changes/22/908722/1
```

Comment:
what is this PR and why not referring to custom zuul change provided by @gtema ?

https://review.opendev.org/c/zuul/zuul/+/859940

Reply:
This pr from https://opendev.org/zuul/zuul/src/commit/81d3c861153ee1fc9709d0fd8e9c2f8bce96ee7a it linked with 9.5.0 version, next major version from 8.2.1

Is this ref should be static?

Reply:
resolved